master/master.go

```diff
@@ -33,6 +37,13 @@ func TerminateSlaves(svc *ec2.EC2, instances []*ec2.Instance) (*ec2.TerminateIns
 	return svc.TerminateInstances(params)
 }
 
+func SendTask(t *task.Task, i *ec2.Instance) (resp *http.Response, err error) {
+	url := "http://" + *i.PublicIpAddress + ":8080/tasks/create"
+	bodyType := "application/json"
+	body, _ := t.ToJson()
```

Comment:
We should panic here because the Task should always be deserialisable. For other errors that could occur, we need to return the error type, like `return (nil, err)`.